### ISSUE TITLE
Defining Local parameters in model

### ISSUE THREAD

**Issue by DaluS** [DaluS/GEMMES#117]
This is I think a very simple issue, in which we forgot just a few lines at the right place. 

It is sometimes practical to define model-specific parameters that do not belong to `def_fields`. The core recognize the existent of `param` section in `logics` dictionary, however, I have not found how to write in a way that the system accept them (mostly problem of lambda calling/expression ?)

For the moment to go around this problem, I added a group in `def_fields` called temporary, however that's not the most beautiful solution !

**Comment by DaluS** [comment]
Something like 

```

import numpy as np

# ---------------------------
# user-defined function order (optional)
_FUNC_ORDER = None


# ---------------------------
# user-defined model
# contains parameters and functions of various types


_LOGICS = {
    'ode': {
        'x': {
            'func': lambda sigma=0, y=0, itself=0: sigma*(y-itself),
            'com': 'X',
            'initial': 0,
        },
        'y': {
            'func': lambda rho=0, x=0, z=0, itself=0: x*(rho-z)-itself,
            'com': 'Y',
            'initial': 1,
        },
        'z': {
            'func': lambda beta=0, x=0, y=0, itself=0: x*y-beta*itself,
            'com': 'Z',
            'initial': 1,
        },
    },
    'statevar': {
    },


### HERE ! ###
    'params': {
        'sigma': {
            'value': 10,
            'definition': 'parameter 1 in Lorenz'},
        'beta': {
            'value': 8/3,
            'definition': 'parameter 2 in Lorenz'},
        'rho': {
            'value': 28,
            'definition': 'parameter 3 in Lorenz'},
    },

###
}


# ---------------------------
# List of presets for specific interesting simulations
_PRESETS = {
    'Canonical example': {
        'fields': {
            'x': np.linspace(1, 1.1, 1),
            'y': 1,
            'z': 1,
            'sigma': 10,
            'rho': 28,
            'beta': 8/3,
        },
        'com': '',
        'plots': [],
    },
}
 

```

**Comment by DaluS** [comment]
Solved in #143
